test/native/integration-test-helpers/rich-text-select-range.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { typeInRichText } from './rich-text-type';
+
+/**
+ * Select a range within a RichText component.
+ *
+ * @param {import('react-test-renderer').ReactTestInstance} richText RichText test instance.
+ * @param {number}                                          start    Selection start position.
+ * @param {number}                                          end      Selection end position.
+ *
+ */
+export const selectRangeInRichText = ( richText, start, end = start ) => {
+	if ( typeof start !== 'number' ) {
+		throw new Error( 'A numerical range start value must be provided.' );
+	}
```

Comment:
In an effort to [mitigate any confusion](https://github.com/WordPress/gutenberg/pull/49804#discussion_r1165601733) as to how this helper functions, I renamed it from `selectTextInRichText` to `selectRangeInRichText`. I also made the `start` parameter required.

Reply:
Sounds great, the name is clearly descriptive to avoid confusion 👍 .